### ISSUE TITLE
OsNet with triplet loss

### ISSUE THREAD

**Issue by Rathna21** [KaiyangZhou/deep-person-reid#310]
Hello, 

I am trying to train Market1501 using triplet loss. 

My accuracy is always zero. 

Here is my config file, 

model:
  name: 'osnet_x1_0'
  pretrained: True

data:
  type: 'image'
  sources: ['market1501']
  targets: ['market1501']
  height: 256
  width: 128
  combineall: False
  transforms: ['random_flip']
  save_dir: 'log/osnet_x1_0_market1501_triplet_cosinelr'

loss:
  name: 'triplet'
  softmax:
    label_smooth: True

train:
  optim: 'amsgrad'
  lr: 0.0002
  max_epoch: 250
  batch_size: 64
  fixbase_epoch: 10
  open_layers: ['classifier']
  lr_scheduler: 'cosine'

test:
  batch_size: 300
  dist_metric: 'euclidean'
  normalize_feature: False
  evaluate: False
  eval_freq: -1
  rerank: False
  visactmap: False
  
sampler:
    train_sampler: 'RandomIdentitySampler'

Please find the screenshot
![triplet_loss](https://user-images.githubusercontent.com/34045027/75989004-c517f680-5ef2-11ea-968c-0df68522548c.png)

Please let me know what is the wrong thing I am doing ?

**Comment by pribadihcr** [comment]
Hi @Rathna21, its resolved?
